setup.py

```diff
@@ -179,6 +180,8 @@
         "pytest-env",
         "Sphinx>4.0.0,<7",
         "testcontainers==4.4.0",
+        "python-keycloak==4.2.2",
+        "firebase-admin>=5.2.0,<6",
```

Comment:
Any reason this was added back?

Reply:
you mean ->  "firebase-admin>=5.2.0,<6", possible that added back due to rebase solving the conflicts.

Reply:
yeah, I removed it in #4359